app/models/book.js

```diff
@@ -1,11 +1,15 @@
 import DS from 'ember-data';
 
 export default DS.Model.extend({
-  name: DS.attr('string'),
-  author: DS.attr('string'),
-  sDescription: DS.attr('string'),
-  lDescription: DS.attr('string'),
-  image: DS.attr('string'),
-  price: DS.attr('decimal'),
-  amount: DS.attr('integer')
-});
\ No newline at end of file
+  name: DS.attr(),
+  author: DS.attr(),
+  sDescription: DS.attr(),
```

Comment:
we talked about the attribute names... these names don't really tell me much you mean description and summary?